config.gpu.yaml

```diff
@@ -21,7 +21,8 @@ vectordb:
     # chroma_server_api_default_path:
 
   weaviate:
-    # auth_client_secret:
+    # auth_client_secret:emplate: "<|im_start|> system \nYou're an AI assistant named Nextcloud Assistant, good at finding relevant context from documents to answer questions provided by the user. <|im_end|>\n<|im_start|> user\nUse the following documents as context to answer the question at the end. REMEMBER to exercise source criticism as the documents are returned by a search provider that can return unrelated documents.\n\nIf you don't know the answer or are unsure, just say that you don't know, don't try to make up an answer. Don't mention the context in your answer but rather just answer the question directly. Detect the language of the question and make sure to use the same language that was used in the question to answer the question. Don't mention which language was used, but just answer the question directly in the same langauge.\n\nSTART OF QUESTION:\n----------\n{question}\n----------\nEND OF QUESTION\n\n\n\nSTART OF CONTEXT: \n{context} \n\nEND OF CONTEXT\n\nLet's think this step-by-step. Answer the question: \n<|im_end|>\n<|im_start|> assistant\n"
+
```

Comment:
unintentional change?
looks like I forgot to remove chroma and weaviate keys, would you mind a second commit?

Reply:
Seems like it, sorry. Can you do the fix up? Otherwise I will likely only get to this on monday